pkg/tempo/config.go

```diff
@@ -61,11 +62,11 @@ func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if err := unmarshal((*plain)(c)); err != nil {
 		return err
 	}
-	return c.Validate()
+	return nil
```

Comment:
Can we just have this changed to `return unmarshal((*plain)(c))` and remove the if above?